.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: '/'
+    schedule:
+      interval: daily
+
```

maybe adding ghactions as well to update release-drafter?
```suggestion
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
```